SpecFlowProject4/StepDefinitions/DynamicControlsStepDefinitions.cs

```diff
@@ -0,0 +1,48 @@
+﻿using ApiTests.Tests.UI;
+using TestProject.Tests.Pages;
+using TestProject.Utils;
+using NUnit.Framework;
+using SpecFlowProject4.StepDefinitions;
+
+namespace DynamicControlsStepDefinitions
+{
+    [Binding]
+    internal class DynamicControlsStepDefinitions : SetUp
+    {
+        readonly DynamicControlsPage dynamicControlsPage = new();
+        readonly BaseTest baseTest = new();
```

Comment:
BaseTest sounds like a weird choice of a name, should be smth more specific, especially since this one looks like a page class